README.md

```diff
@@ -89,16 +89,15 @@ associated with each stack frame by setting the ```--locals``` flag.
 
 ### Why do we need another Python profiler?
 
-This project aims to let you profile and debug any running Python program, even if the program is
+This project aims to let your profile and debug any running Python program, even if the program is
```

Comment:
`profile` here is used as a verb, and not referencing to _your_ profile. It lets you <verb>profile</verb> your code. So this actually isn't a correct revision